staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -127,8 +128,9 @@ type LogsOptions struct {
 	containerNameFromRefSpecRegexp *regexp.Regexp
 }
 
-func NewLogsOptions(streams genericclioptions.IOStreams, allContainers bool) *LogsOptions {
-	return &LogsOptions{
+// NewLogsOptions returns a default LogOptions.
+func NewLogsOptions(streams genericclioptions.IOStreams, allContainers bool) *Options {
+	return &Options{
```

Comment:
Change to `NewOptions` may be better

Reply:
Thanks! changed `NewLogsOptions` to `NewOptions`